ckanext/qdes_schema/templates/package/read_base.html

```diff
@@ -11,7 +11,7 @@
   {% endif %}
   {{ h.build_nav_icon(dataset_type ~ '.groups', _('Groups'), id=id_or_name, icon='users') }}
 
-  {% if h.check_access('package_update', {'id':pkg.id }) %}
+  {% if h.check_access('package_update', {'id':pkg.id }) and dataset_type == 'dataset' %}
```

Comment:
Move logic around so dataset_type is first. There is no point check user access if it is not the right dataset_type